src/fhirtypes/StructureDefinition.ts

```diff
@@ -60,11 +62,13 @@ export class StructureDefinition {
    * A base clone of the Structure Definition from before any rules were applied
    */
   private _baseStructureDefinition: StructureDefinition;
+  private _structureDefinitionStructureDefinition: StructureDefinition;
```

Comment:
I don't know about you, but I'd be comfortable with `sdStructureDefinition`.  It's shorter and less repetitive.